documentation/jetty/modules/operations-guide/pages/modules/standard.adoc

```diff
@@ -129,6 +129,19 @@ The module properties are:
 include::{jetty-home}/modules/debuglog.mod[tags=documentation]
 ----
 
+[[dos]]
+== Module `dos`
+
+The `dos` module installs the `org.eclipse.jetty.server.handler.DoSHandler` at the root of the `Handler` tree.
+
+The `DoSHandler` limits the rate of requests per remote client, as explained in xref:programming-guide:server/http.adoc#handler-use-dos[this section].
```

Comment:
Should we say that there is only so much a server can do to protect itself from DOS and that true solutions are often better implemented in load balancers and other data center network infrastructure?

Reply:
Done.